### PR TITLE
create an early program which input numbers from user

### PR DESCRIPTION
**Pull request by ArtomBusniuk**

### CODE REVIEW

File: src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
-            //TIP Press <shortcut actionId="Debug"/> to start debugging your code. We have set one <icon src="AllIcons.Debugger.Db_set_breakpoint"/> breakpoint
-            // for you, but you can always add more by pressing <shortcut actionId="ToggleLineBreakpoint"/>.
-            System.out.println("i = " + i);
+        Scanner number = new Scanner(System.in);
+        System.out.println("Wpisz 1 albo 2:");
+        String  name = number.nextLine();
+        System.out.println(name);
+        if ("1".equals (name)) {
+            System.out.println("Wszedles");
+        } else {
```
Comment (brtkbaranowsky) [inline src/main/java/org/example/Main.java]: Get rid of magic values - final variables with better description

File: src/main/java/org/example/Main.java [hunk 2]
```diff
@@ -1,20 +1,16 @@
...
-        System.out.printf("Hello and welcome!");
-
-        for (int i = 1; i <= 5; i++) {
-            //TIP Press <shortcut actionId="Debug"/> to start debugging your code. We have set one <icon src="AllIcons.Debugger.Db_set_breakpoint"/> breakpoint
-            // for you, but you can always add more by pressing <shortcut actionId="ToggleLineBreakpoint"/>.
-            System.out.println("i = " + i);
+        Scanner number = new Scanner(System.in);
+        System.out.println("Wpisz 1 albo 2:");
+        String  name = number.nextLine();
+        System.out.println(name);
```
Comment (brtkbaranowsky) [inline src/main/java/org/example/Main.java]: get rid of System, use static import

File: src/main/java/org/example/Main.java [hunk 3]
```diff
@@ -1,20 +1,16 @@
...
+        if ("1".equals (name)) {
+            System.out.println("Wszedles");
+        } else {
+            System.out.println("Nie prawidlowe haslo");
```
Comment (brtkbaranowsky) [inline src/main/java/org/example/Main.java]: ditto

File: src/main/java/org/example/Main.java (hunk 1 above)
Comment (ArtomBusniuk) [reply src/main/java/org/example/Main.java]: Changed as requested

**Pull request merged**